### DISCUSSION TITLE
Moving to the new .NET Foundation CLA service

### DISCUSSION THREAD

**Discussion by dnfadmin** [nanoframework/Home#1234] [General]
Hello everyone
The .NET Foundation is transitioning to a new CLA service and I'm posting here to let you know it's happening. Some background info:

The new CLA service is based off of https://github.com/microsoft/ContributorLicenseAgreement, except it uses the dotnet-policy-service app

- Current contributors do not need to sign the CLA again
- The CLA agreement itself is not changing
- As part of the install/configuration process, the following work items will be completed

1. Create a .github repo if it doesn't exist already
2. Install the dotnet-policy-service app
3. Create PRs for the platformcontext.xml and cla.yml files
4. Disable the existing CLA webhook
5. Push the PRs to enable the new CLA service

That's all there is to it. I'll update this issue once the work is complete

Thanks
Chris
.NET Foundation Operations team

**Comment by josesimoes** [comment]
Noted. Please let me know if you need anything from our end.

**Comment by dnfadmin** [comment]
Hi @josesimoes  - nope, no help needed - this work is done :-)
